test/models/organization_test.rb

```diff
@@ -1,6 +1,64 @@
 require 'test_helper'
 require 'models/shared/taxonomies_base_test'
+require 'rfauxfactory'
 
 class OrganizationTest < ActiveSupport::TestCase
   include TaxonomiesBaseTest
+
+  # List of valid organization names.
+  def valid_org_name_list
+    # Note: The maximum allowed length of org name is 242 only. This is an
+    # intended behavior (Also note that 255 is the standard across other
+    # entities.)
+    [
+      RFauxFactory.gen_alpha(1),
+      RFauxFactory.gen_alpha(242),
+      *RFauxFactory.gen_strings(1..242, exclude: [:html]).values,
+      RFauxFactory.gen_html(rand((1..217)))
+    ]
+  end
+
+  # List of invalid organization names.
+  def invalid_org_name_list
+    [
+      '',
+      ' ',
+      "\t",
+      *RFauxFactory.gen_strings(243).values
+    ]
+  end
+
+  test "create with multi names" do
+    valid_org_name_list.each do |name|
+      organization = FactoryBot.create(:organization, :name => name)
+      organization.reload
+      assert_equal organization.name, name
+    end
+  end
+
+  test "should not create with invalid names" do
+    invalid_org_name_list.each do |name|
+      organization = Organization.new(:name => name)
```

Comment:
it would be better to use `FactoryBot.build(:organization, :name => name)` since that ensures that the new org object is valid (except for the invalid name). If org would have other attributes which presence is required, or other name validations, the test could pass without catching the name validations we're testing.